projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -73,276 +72,293 @@ describe('DatagridComponent', () => {
             declarations: [HostWithDatagridComponent, DatagridDetailsComponent, DatagridDetailsPaneComponent],
         }).compileComponents();
 
-        this.finder = new WidgetFinder(HostWithDatagridComponent);
-        this.vcdDatagrid = this.finder.find(GridWithBoldRenderer);
-        this.clrGridWidget = this.vcdDatagrid.clrDatagrid;
-        this.component = this.finder.find(VcdDatagridWidgetObject).component as DatagridComponent<MockRecord, RecordId>;
+        this.finder = new AngularWidgetObjectFinder(HostWithDatagridComponent);
+        // @ts-ignore
+        this.vcdDatagrid = this.finder.find(VcdDatagridWidgetObject);
+        this.clrGridWidget = this.vcdDatagrid.findClrDatagrid();
+        this.component = this.vcdDatagrid.unwrap().elements[0].componentInstance as DatagridComponent<
```

Comment:
Instead of doing this, we should rely on using this.finder.hostComponent to get access to all the Angular Components. (I think this should work, but let me know if it doesn't).